packages/ui/src/components/Footer/Footer.tsx

```diff
@@ -54,100 +54,128 @@ const socials: readonly SocialLinks[] = [
 
 export const Footer = () => {
   return (
-    <footer className="main-container">
-      <div className="footer">
-        <nav aria-label="social" className="px-4">
-          {socials.map((social) => {
-            const Icon = Icons[social.icon];
-            return (
-              <TooltipProvider delayDuration={30} key={social.href}>
-                <Tooltip>
-                  <TooltipTrigger>
-                    <a aria-label={social.alt} href={social.href} className="group" target="_blank">
-                      {Icon ? <Icon className="h-16 w-16 md:h-20 md:w-20" /> : null}
-                    </a>
-                  </TooltipTrigger>
-                  <TooltipContent>
-                    <p>{social.tooltipText}</p>
-                  </TooltipContent>
-                </Tooltip>
-              </TooltipProvider>
-            );
-          })}
-        </nav>
-        <nav aria-label="quick links" className="h-20 divide-x">
-          {links.map((link) => (
-            <TooltipProvider delayDuration={30} key={link.href}>
+    <footer className=" absolute bottom-0 z-[999] w-full px-[30px] pb-14">
```

Comment:
@rohann06 on the LessonPage the footer is broken, can you check it? maybe because of the `absolute` 

![image](https://github.com/user-attachments/assets/b0db7211-7bbf-4581-81f6-69c33444752d)